Makefile

```diff
@@ -33,35 +33,35 @@ define LDFLAGS
 "
 endef
 
-BUILDTAGS = containers_image_ostree_stub exclude_graphdriver_devicemapper exclude_graphdriver_btrfs containers_image_openpgp
+BUILDFLAGS = -tags "netgo containers_image_ostree_stub exclude_graphdriver_devicemapper exclude_graphdriver_btrfs containers_image_openpgp" -installsuffix netgo
 
 .PHONY: test
 test:
-	go test -tags "$(BUILDTAGS)" ./pkg/... ./cmd/... ./ffi/... -coverprofile cover.out
+	go test $(BUILDFLAGS) ./pkg/... ./cmd/... ./ffi/... -coverprofile cover.out
 
 .PHONY: integration-cli
 integration-cli:
 	go build -o bin/kots-integration ./integration
 
 .PHONY: ci-test
 ci-test:
-	go test -tags "$(BUILDTAGS)" ./pkg/... ./cmd/... ./ffi/... ./integration/... -coverprofile cover.out
+	go test $(BUILDFLAGS) ./pkg/... ./cmd/... ./ffi/... ./integration/... -coverprofile cover.out
 
 .PHONY: kots
 kots: fmt vet
-	go build ${LDFLAGS} -o bin/kots -tags "$(BUILDTAGS)" github.com/replicatedhq/kots/cmd/kots
+	CGOENABLED=0 go build ${LDFLAGS} -o bin/kots $(BUILDFLAGS) github.com/replicatedhq/kots/cmd/kots
 
 .PHONY: ffi
 ffi: fmt vet
-	go build ${LDFLAGS} -o bin/kots.so -tags "$(BUILDTAGS)" -buildmode=c-shared ./ffi/...
+	go build ${LDFLAGS} -o bin/kots.so -buildmode=c-shared ./ffi/...
```

Comment:
No buildflags here?

Reply:
just making it match goreleaser. im not sure what is correct